packages/auth/src/grant/routes.test.ts

```diff
@@ -938,6 +938,63 @@ describe('Grant Routes', (): void => {
     })
   })
 
+  describe('Deny interaction', (): void => {
+    test('Cannot deny interaction without interaction id', async (): Promise<void> => {
+      const ctx = createContext(
+        {
+          headers: {
+            Accept: 'application/json',
+            'Content-Type': 'application/json'
+          }
+        },
+        {}
+      )
+
+      ctx.session.interactId = grant.interactId
```

Comment:
```suggestion
          },
          session: {
            interactId: grant.interactId
          }
        },
        {}
      )
```
`session` can be passed in [`httpMocks.RequestOptions`](https://github.com/howardabrams/node-mocks-http#createrequest)

`ctx.session = req.session` would need to be added in [`createContext`](https://github.com/interledger/rafiki/blob/main/packages/auth/src/tests/context.ts)